test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/Dockerfile

```diff
@@ -2,8 +2,8 @@
 # This should be run from the root of the repo:
 # docker build --file test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/Dockerfile .
 
-ARG BUILD_SDK_VERSION=8.0
-ARG TEST_SDK_VERSION=8.0
+ARG BUILD_SDK_VERSION=9.0
```

Comment:
Not sure how to add comment on parts not shown on the diff. I'll just add here.

Line 16 of this file is still set to net8.0 in this file. Does it need update?

Line 31 and the part of Line 32 after `||` can be removed since net6.0 is removed.

Reply:
Good catch! These versions are used to get the Docker images . The value in `PUBLISH_FRAMEWORK` is used with `dotnet publish`.

`PUBLISH_FRAMEWORK` needs to be set to 9.0.

Reply:
There's one more:

> Line 31 and the part of Line 32 after `||` can be removed since net6.0 is removed.